lte/gateway/python/magma/pipelined/app/uplink_bridge.py

```diff
@@ -202,3 +245,16 @@ def _set_sgi_static_ip(self):
             self.logger.debug("SGi ip address config: [%s]", set_ip_cmd)
         except subprocess.SubprocessError as e:
             self.logger.warning("Error while setting SGi IP: %s", e)
+
+    def _setup_vlan_pop_dev(self):
```

Comment:
Wait this is only used in testing? Lets move it to the test file then, no reason to do this for production datapath

Reply:
the port number are used for deterministic test.